plugins/chain_plugin/chain_plugin.cpp

```diff
@@ -280,11 +280,14 @@ read_only::get_info_results read_only::get_info(const read_only::get_info_params
 }
 
 abi_def get_abi( const chain_controller& db, const name& account ) {
-   const auto& d = db.get_database();
-   const auto& code_accnt  = d.get<account_object,by_name>( account );
-
+   const auto &d = db.get_database();
+   const account_object *code_accnt = nullptr;
+   try {
+      code_accnt = &d.get<account_object, by_name>(account);
```

Comment:
get returns a reference, therefore it can never be null. It will throw an exception if not found.  Perhaps you want d.find<...

Reply:
Right. Thanks for pointing it out! PR Updated.